src/watchmaker/workers/salt.py

```diff
@@ -477,14 +489,16 @@ def process_states(self, states, exclude):
                 Comma-separated string of salt states to execute. Accepts two
                 special keywords (case-insensitive):
 
-                - ``none``: Do not apply any salt states.
+                - ``null``: Do not apply any salt states.
```

Comment:
This one may need special handling. I think this impacts the cli usage of `--salt-states None`. Does that still work, and just outputs the log warning?

Reply:
`null` is standard YAML but for the CLI, I think that `"None"` or `"none"` should be valid and not deprecated. But, we won't know whether it comes from YAML or the CLI.